pylabnet/hardware/ni_daqs/nidaqmx_card.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+
+"""
+This file contains the pylabnet Hardware module class for a generic NI DAQ mx card.
+
+"""
+
+import nidaqmx  
+from pylabnet.utils.logging.logger import LogHandler
+
+
+class Ni_Daq_Mx_Card():
+
+    def __init__(self, device_name, logger=None):
+        """Instantiate NI DAQ mx card
+
+        :device_name: Name of NI DAQ mx card, as displayed in the measurement and automation explorer
+        """
+
+        # Device name
+        self.dev = device_name
+
+        # Log
+        self.log = LogHandler(logger=logger)
+
+        # Try to get infos of DAQ device to verify connection
+        try:
+            ni_daq_device = nidaqmx.system.device.Device(name = device_name)
+            self.log.info(
+                "Successfully connected to NI DAQ '{device_name}' (type: {product_type}) \n".format\
+                    (device_name = ni_daq_device.name, product_type=ni_daq_device.product_type)
+                )
+        
+        # If failed, provide info about connected DAQs
+        except:
```

Comment:
@cknaut Exceptions. Here we should probably discuss it more to decide what is the best option.
I would suggest not only log error, but also produce an actual exception using ```raise ExceptionName```.